Tests/SwiftGRPCTests/ServerThrowingTests.swift

```diff
@@ -52,15 +55,37 @@ class ServerThrowingTests: BasicEchoTestCase {
 }
 
 extension ServerThrowingTests {
-  func testServerThrowsUnary() {
+  func testServerThrowsUnary() throws {
     do {
       let result = try client.get(Echo_EchoRequest(text: "foo")).text
       XCTFail("should have thrown, received \(result) instead")
-    } catch {
-      guard case let .callError(callResult) = error as! RPCError
+      return
```

Comment:
No need for this return since `XCTFail` should end the test

Reply:
XCTFail generates a failure but does not exit. In fact, the compiler would have warned about those return statements otherwise.

Reply:
Aha, I was thinking about `continueAfterFailure` - for some reason I thought it defaulted to `true`, but turns out it [defaults to `false`](https://developer.apple.com/documentation/xctest/xctestcase/1496260-continueafterfailure). My bad 🙃